snap/info_snap_yaml.go

```diff
@@ -239,10 +239,6 @@ func infoSkeletonFromSnapYaml(y snapYaml) *Info {
 	if y.Type != "" {
 		typ = y.Type
 	}
-	// TODO: once we have epochs transition to the snapd type for real
-	if y.Name == "snapd" {
```

Comment:
Don't we still need this one before https://github.com/snapcore/snapd/pull/7092 is merged? It seems like we can merge this even without 7092 if we keep this assignment, no?

Reply:
yes, I think it would be simpler to not do this here but to do it in 7092. It also means I think we don't need the spread test hack here.

Reply:
I moved this code to 7092 now.